_data/sponsor_logos_l.csv

```diff
@@ -1,3 +1,4 @@
 logo,name,url,last_payment,all_time,since,level
 sponsors/84.png,84codes,https://www.84codes.com/,"€22,000","€369,000","Apr 1, 2018",5000
 manas-orange.svg,Manas.Tech,https://manas.tech/,"$5,000","$1,300,000","Jun 19, 2009",5000
+,Austin Robert Bales,https://arbales.me/,"$1,234","$1,234","May 4, 2024",1000
```

Comment:
This broke the expectation that big logo sponsors actually *have* a logo.

![grafik](https://github.com/crystal-lang/crystal-website/assets/466378/6b9eda6f-6ae9-47fb-82b0-42a4e54c5284)

This is now breaking the link checker: https://github.com/crystal-lang/crystal-website/actions/runs/9228044592/job/25391263090. Apparently these CI jobs were not run on the PR for some reason.